packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -260,6 +260,55 @@ export const prepareRoundsManagerKit = baggage =>
           });
         },
 
+        /**
+         * @param {bigint} roundId
+         * @param {Timestamp} blockTimestamp
+         */
+        initializeVstorage(roundId, blockTimestamp) {
+          const {
+            latestRoundPublisher,
+            minSubmissionCount,
+            maxSubmissionCount,
+            timeout,
+          } = this.state;
+          const { helper } = this.facets;
+          helper.updateTimedOutRoundInfo(subtract(roundId, 1), blockTimestamp);
+
+          this.state.reportingRoundId = roundId;
+          const rounds = makeScalarBigMapStore('rounds', { durable: true });
+          const details = makeScalarBigMapStore('details', { durable: true });
+          // @ts-expect-error append-only except for reset
+          this.state.rounds = rounds;
+          // @ts-expect-error append-only except for reset
+          this.state.details = details;
+
+          details.init(
```

Comment:
I'm not confident that I know the invariants of the `state.details` map stores. @michaelfig or @turadg are you able to tell from a quick look that this is OK?